src/main/java/ch/heigvd/amt/stack/infrastructure/persistence/database/JdbcRepository.java

```diff
@@ -27,7 +27,7 @@
             "CREATE TABLE IF NOT EXISTS Session" +
                     "( idSession VARCHAR PRIMARY KEY" +
                     ", idxCredential VARCHAR" +
-                    ", tag VARCHAR(50)" +
+                    ", tag VARCHAR(50) PRIMARY KEY " +
```

Comment:
The `tag` is not (and should not be) part of the primary key. They may have a unique clause. This might be a reason for which e2e tests fail.